spec/requests/api/custom_attributes_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe "Custom Attributes API" do
+  it "can delete a custom attribute through its nested URI" do
+    vm = FactoryGirl.create(:vm_vmware)
+    custom_attribute = FactoryGirl.create(:custom_attribute, :resource => vm)
+    api_basic_authorize
+
+    run_delete("#{vms_url(vm.id)}/custom_attributes/#{custom_attribute.id}")
+
+    expect(response).to have_http_status(:no_content)
```

Comment:
please add an expectation that the custom_attribute no longer exists in Model. Thanks.